apps/cms/src/payload.config.ts

```diff
@@ -1,10 +1,13 @@
 import { buildConfig } from 'payload/config';
+import { cloudStorage } from '@payloadcms/plugin-cloud-storage';
+import { s3Adapter } from '@payloadcms/plugin-cloud-storage/s3';
```

Comment:
can we alias this import to `createS3Adapter` ? So that it doesn't conflict with `s3adapted` (below)